LICENSE

```diff
@@ -0,0 +1,19 @@
+Copyright (c) 2017 Contentful GmbH
```

Comment:
Is this the MIT license? Can we include that line `The MIT License (MIT)` used in the other contentful repositories?

Reply:
Another thing every repo seems to do different 😆 Adding the heading makes sense though, most people don't just recognize licenses.

(This comment doesn't show up as outdated because the line stayed intact)